pkg/metrics/metrics.scss

```diff
@@ -233,6 +233,24 @@
       }
     }
   }
+
+  .all-disks-no-gap {
+    // don't center the "View all disks" button
+    button {
+      padding-left: 0;
+      width: fit-content;
+    }
+  }
+}
+
+.pf-c-popover {
+  .disks-nowrap-white-space {
+    td[data-label="Device"],
+    td[data-label="Read"],
+    td[data-label="Write"] {
```

Comment:
Would this work on translated page?

Reply:
Nope, it wouldn't. The css should be now fixed here and in the network card.